src/ApolloContext.ts

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ApolloClient from 'apollo-client';
+import { DocumentNode } from 'graphql';
+
+export interface ApolloContextValue {
+  client?: ApolloClient<Object>;
+  operations?: Map<string, { query: DocumentNode; variables: any }>;
+}
+
+export const ApolloContext = React.createContext
```

Comment:
I manually ran the tests with `false && React.createContext` here instead, and they still pass. Any thoughts on a good way to run tests automatically both ways?

Reply:
As long as the tests are passing, we should be golden with your approach @benjamn (since the Preact tests passing prove that the legacy API fallback is working). This is great - thanks!